engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -184,4 +185,16 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
         }
       }
     }
-}
\ No newline at end of file
+
+  def backendRoute =
+    path("workflows" / Segment / "backends") { version =>
+      get {
+        val backends = BackendConfiguration.AllBackendEntries map { _.name }
+        complete {
+          JsObject(Map("backends" -> backends.toJson))
+        }
+      }
+    }
+
+  val backends = BackendConfiguration.AllBackendEntries map { _.name } toJson
```

Comment:
I lied. I'm going to do the above, as I'd rather be lazy on the Centaur side for now and the above will enable just that